docs/01-Using-Fleet/FAQ.md

```diff
@@ -141,7 +141,7 @@ You can also do this by setting the `targets` field in the [YAML configuration f
 
 ## How do I automatically assign a host to a team when it enrolls with Fleet?
 
-[Team enroll secrets](https://github.com/fleetdm/fleet/blob/main/docs/01-Using-Fleet/10-Teams.md#enroll-hosts-to-a-team) allow you to automatically assign a host to a team.
+[Team enroll secrets](./01-Using-Fleet/10-Teams.md#enroll-hosts-to-a-team) allow you to automatically assign a host to a team.
```

Comment:
Was this one missed? Should it also point to `https://fleetdm.com/docs/...`?

Reply:
I'm using a relative link because the page is inside the `/docs` folder. This keeps the link working on both fleetdm.com and github.com.

Reply:
This is failing the links check though https://github.com/fleetdm/fleet/runs/5148572897?check_suite_focus=true

Reply:
Thanks for catching that, the link has been updated.